R/sp_get_fivenumsum.R

```diff
@@ -0,0 +1,75 @@
+#' Calcualte a 5 numbeer summary for all types of variables
```

Comment:
Typos, should be: `Calculate a 5 number summary`

Reply:
:-)